target/scripts/helpers/error.sh

```diff
@@ -62,13 +62,11 @@ function dms_panic__no_file { dms_panic 'no-file' "${1}" "${2}"; }
 function dms_panic__misconfigured { dms_panic 'misconfigured' "${1}" "${2}"; }
 function dms_panic__invalid_value { dms_panic 'invalid-value' "${1}" "${2}"; }
 
-# Call this method when you want to panic (emit a 'FATAL' log level error, and exit uncleanly).
+# Call this method when you want to panic (emit an 'ERROR' log and exit uncleanly).
 # `dms_panic` methods should be preferred if your failure type is supported.
 function _shutdown
 {
-  local FATAL_ERROR_MESSAGE=$1
-
-  _notify 'fatal' "${FATAL_ERROR_MESSAGE}"
-  _notify 'err' "Shutting down.."
-  kill 1
+  _notify 'error' "${1}"
+  _notify 'error' 'Shutting down...'
+  exit 1
 }
```

Comment:
Was dropping `kill 1` intentional here?

Reply:
No :D

Reply:
Done in [809b2b4](https://github.com/docker-mailserver/docker-mailserver/pull/2486/commits/809b2b42a9d4bc9340e91a5a29883b19b99d574e)